maple/__main__.py

```diff
@@ -0,0 +1,168 @@
+import argparse
+import code
+import csv
+from collections import defaultdict
+from datetime import datetime
+from pathlib import Path
+
+import pandas as pd
+from tqdm import tqdm
+
+from maple.classification import regex_classification
+from maple.db import connect
+from maple.types import Action, ActionType, Bill, Branch
+
+
+def parse_bills(bills_file: Path) -> list[Bill]:
```

Comment:
Would be nice to add docstrings

Reply:
Ya. My propensity to write docstrings is inversely proportional to the length of the function, so I probably left out quite a few.